src/libPOW/pow.cpp

```diff
@@ -34,46 +34,17 @@ POW::POW()
     currentBlockNum = 0;
     ethash_light_client = EthashLightNew(
         0); // TODO: Do we still need this? Can we call it at mediator?
+
+#ifndef IS_LOOKUP_NODE
```

Comment:
I'm not creating troubles here but I don't really like using `IS_LOOKUP_NODE` macro here. Maybe the discussion was not clear and transparent before. Let's revisit this issue again. My suggestion is gradually removing the uses of this macro (or at least not adding any new ones 'cause it's adding up the difficulty). We don't actually enjoy too many benefits from it but suffer a lot in sorting out the build system (CMake) and CI/CD workflow.

Reply:
If don't use this MACRO, is there any other variable can differentiate the normal node or look_up node?

Reply:
I think we don't need to fix this issue here. I'll create a separate issue and you can fix all the MACRO in another PR. @ShengguangXiao